### PR TITLE
Sc 1599 min video player

### PR DESCRIPTION
**Pull request by adrianjost**
replaced the OpenHPI video player with the native HTML5 player. This reduces the loaded overhead and the browser can decide which data it will load (`preload="auto"`).

## Allgemein
- [x] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-1599

### CODE REVIEW

**Comment by adrianjost** [comment]
@tofixx @schurigh ping

**Comment by malcm** [comment]
@adrianjost good point but keep in mind native HTML5 player does not have HLS support in all browsers. If needed in future

**Comment by adrianjost** [comment]
@malcm true, but it was common sense that we don't need it for our simple use cases.

**Comment by malcm** [comment]
@adrianjost ok good. I just remember that @janrenz wants to have it in the schul-cloud

**Comment by adrianjost** [comment]
@malcm When I discussed this with him the last time (a month ago) he liked the idea of removing it for performance reasons 😅

**Comment by malcm** [comment]
👍

File: static/styles/homepage/homepage.scss [hunk 1]
```diff
@@ -27,3 +27,7 @@ h1.section-title{
         font-size: 36px;
     }
 }
+
+video{
```
Comment (tofixx) [inline static/styles/homepage/homepage.scss]: lint

File: static/styles/lib/base.scss [hunk 2]
```diff
@@ -89,6 +89,9 @@ p.small {
 label {
 	font-weight: bold;
 }
+video, audio {
+	width:100%;
```
Comment (tofixx) [inline static/styles/lib/base.scss]: lint

**Comment by tofixx** [comment]
@adrianjost please fix the height issue and resolve conflicts that we may include this pr in the next release asap

**Comment by adrianjost** [comment]
@tofixx adjusted it and should work in all modern browsers (https://caniuse.com/#search=object-fit)

**Pull request merged**